.travis.yml

```diff
@@ -63,11 +63,12 @@ rvm:
   - 2.2.8
   - 2.3.5
   - 2.4.2
+  - 2.5.0
   - ruby-head
 
 matrix:
   include:
-    - rvm: 2.4.2
+    - rvm: 2.5.0
```

Comment:
Do you think we should duplicate this block so we're testing `av:ujs` on both 2.4 and 2.5?

Reply:
I think this is good; that test is limited to a single version because it's mostly testing JS behaviour, not anything special about 2.4+.